### PR TITLE
Add to README.md's "Parsers written with nom"

### PR DESCRIPTION
**Pull request by adamchalmers**
Adding my crate, [fountain](https://github.com/adamchalmers/fountain-rs) to the list of Parsers Written With Nom in the README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -182,7 +182,8 @@ nom parsers are for:
 - [x] **descriptive errors**: the parsers can aggregate a list of error codes with pointers to the incriminated input slice. Those error lists can be pattern matched to provide useful messages.
 - [x] **custom error types**: you can provide a specific type to improve errors returned by parsers
 - [x] **safe parsing**: nom leverages Rust's safe memory handling and powerful types, and parsers are routinely fuzzed and tested with real world data. So far, the only flaws found by fuzzing were in code written outside of nom
-- [x] **speed**: benchmarks have shown that nom parsers often outperform many parser combinators library like Parsec and attoparsec, some regular expression engines and even handwritten C parsers
+- [x] **speed**: bench
```
Comment (Hywan) [inline README.md]: Why this new line?

Comment (adamchalmers) [reply README.md]: Oops. That was a mistake.

Comment (adamchalmers) [reply README.md]: Fixed!

**Comment by adamchalmers** [comment]
@Hywan Hi! Just bumping this. I've fixed the newline.

**Comment by Geal** [comment]
done :)

**Pull request merged**